src/core/error/message-helpers.js

```diff
@@ -41,3 +41,21 @@ export function elementStringOrPassThru(val) {
   }
   return val;
 }
+
+/**
+ * Tests if an error message contains the user sentinel.
+ * @param {string} message
+ * @return {boolean} Whether this message was a user error.
+ */
+export function isUserErrorMessage(message) {
+  return message.indexOf(USER_ERROR_SENTINEL) >= 0;
```

Comment:
```suggestion
  return message.includes(USER_ERROR_SENTINEL);
```

Reply:
String.includes is banned

Reply:
Ohhh, I mixed this with `Array.prototype.includes`. We can unban this, since IE11 is being deprecated (and ESM has full support)

Reply:
Will do in follow-up